tests/units/modules/clustering/kubevirt/test_kubevirt_raw.py

```diff
@@ -0,0 +1,66 @@
+import json
+import sys
+import pytest
+
+from ansible.compat.tests.mock import patch, MagicMock
+from ansible.module_utils import basic
+from ansible.module_utils._text import to_bytes
+
+from openshift.dynamic import Resource
+
+# FIXME: paths/imports should be fixed before submitting a PR to Ansible
+sys.path.append('lib/ansible/modules/clustering/kubevirt')
+
+import kubevirt_raw as mymodule
+
+
+def set_module_args(args):
```

Comment:
Please update the PR and use the utils module and remove both methods and exception.